docs: updating percentage() docs (#19101)
newrelic/docs-website: src/content/docs/nrql/nrql-syntax-clauses-functions.mdx

  >
    Use the `percentage()` function to return the percentage of a target data set that matches some condition.

>     This function expects exactly two arguments (arguments after the first two will be ignored). The first argument requires an [aggregator function](#functions) against the desired attribute. If the attribute is not numeric, this function returns a value of 100%. The second argument requires a where clause.

Reviewer: ```suggestion
    This function expects exactly two arguments (arguments after the first two are ignored). The first argument requires an [aggregator function](#functions) against the desired attribute. If the attribute is not numeric, this function returns a value of 100%. The second argument requires a `WHERE` clause.
```
